pkg/oomstore/types/feature.go

```diff
@@ -87,6 +87,32 @@ func (l FeatureList) Find(find func(*Feature) bool) *Feature {
 	return nil
 }
 
+func (l FeatureList) GroupIDs() []int {
+	ids := []int{}
+	for _, f := range l {
+		ids = append(ids, f.GroupID)
+	}
+	return ids
+}
+
+func (l FeatureList) Any(predict func(*Feature) bool) bool {
```

Comment:
Do you mean `predicate`?

Reply:
yes, would predicate be more appropriate?

Reply:
yes that works

Reply:
replaced `predict` with `predicate`